.pre-commit-config.yaml

```diff
@@ -55,3 +55,7 @@ repos:
         pass_filenames: true
         files: ^src/.*\.py$
         exclude: ^src/snowflake/cli/app/telemetry.py$
+      - id: dependencies-sync
+        name: "Copy dependencies from pyproject.toml to requirements.txt"
+        language: system
+        entry: python .snyk/dependency-sync.py
```

Comment:
Add `files` to run this only on pyproject.toml change, WDYT?

Reply:
Good idea, added